icons/wolframlanguage.svg

```diff
@@ -1 +1 @@
\ No newline at end of file
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Wolfram Language icon</title><path d="M14.3349 12.4314c-.552.149-.615.18-1.164.337-.589-.337-1.1069-.526-1.5949-.463.057-.306.151-.525.245-.8 1.036-.151 1.805.4 2.5139.926zm5.3077 7.2006a10.7954 10.7954 0 01-1.9069 1.5209h-.006l-3.7018-4.6138 3.0698.336s1.471.151 1.807 0c.308-.117 1.9628-2.4488 1.9628-2.4488s-4.7587-3.0089-6.3836-4.6658c.188-2.7939-.213-6.5087-.213-6.4787-1.193 1.195-1.35 1.383-2.5439 2.4889-.52-1.688-.769-2.2689-1.289-3.9578-1.5678 1.289-2.7628 3.4638-3.6197 6.0157a12.2894 12.2894 0 00-.551.656c-.112.157-.23.313-.344.475-.231.326-.45.6749-.663 1.0319-.15.257-.299.52-.438.788-.063.112-.125.231-.188.35-.9129 1.7878-1.6758 3.7887-2.3388 5.6036a10.8244 10.8244 0 01-1.091-4.7337c0-2.8629 1.138-5.6137 3.1639-7.6396C6.3943 2.3338 9.1371 1.2009 12 1.2009a10.7994 10.7994 0 017.6416 3.1579 10.8304 10.8304 0 013.1558 7.6396 10.8134 10.8134 0 01-3.1548 7.6336M12 6.6907c.832-.801.951-.92 1.751-1.689.063 1.532.031 2.333-.063 4.2028-.463-.458-1.381-1.044-1.381-1.044S12.126 7.0907 12 6.6908m3.8338 15.4631C9.2181 24.5468 4.4364 20.14 3.4174 18.602c.006-.014.006-.027.006-.039.92-3.8888 2.058-8.5345 3.8838-9.9095.955-1.6559 1.231-4.1137 2.9429-5.4017.432 1.288 1.107 3.9578 1.57 5.2458 2.0248 2.0259 5.0866 4.5457 7.1455 5.5897.212.12.489.981.489.981l-.825 1.0379-8.8345-.887c-.2-.02-.394-.028-.594-.028-.569 0-1.1499.073-1.8329.18.432-1.071 1.35-1.9359 1.35-1.9359s-.855-.519-1.505-.605c.187-.432.681-.989.799-1.138-.243.087-2.0249.888-2.2069 1.563.857.214 1.469.487 1.469.487s-.949.957-1.131 2.6118c0 0 2.8199-.431 4.9387-.153.063.03.094.03.125.03l1.102.031 3.5088 5.8408.027.045a.012.012 0 01-.011.006m4.6518-18.639A12.0194 12.0194 0 0012 0C8.8182 0 5.7683 1.27 3.5164 3.5148 1.2726 5.7657.0026 8.8165.0026 11.9994c0 3.1828 1.27 6.2347 3.5128 8.4775a11.9794 11.9794 0 0016.9702 0c2.2508-2.2428 3.5118-5.2957 3.5118-8.4775 0-3.1819-1.261-6.2337-3.5118-8.4846z"/></svg>
```

Comment:
LGTM! 🐺🔤
Any chance you can get it down to 3 decimals?

Reply:
@adamrusted Done 🐺